services/cadip/config/cadip_ws_config.yaml

```diff
@@ -765,121 +718,116 @@ ins_session:
       collection: Sentinel1
   search:
     type: ODataV4Search
-    api_endpoint: "http://127.0.0.1:5000/Sessions"
```

Comment:
@agrosu1978 @Padeanu why the port was 5000 instead of 5100 for ins_session ?

Reply:
Same